howso/visuals/visuals.py

```diff
@@ -662,3 +663,74 @@ def plot_fairness_disparity(
         fig.update_xaxes(tickangle=x_tickangle)
 
     return fig
+
+
+def compose_figures(
+    figures: List[go.Figure],
+    rows: int,
+    cols: int,
+    **make_subplots_kwargs,
```

Comment:
should we put something in the docstrings, about these kwargs being passed to make_subplots?

Reply:
NIT, the name of the function is named here so I'm good either way